src/collective/casestudy/profiles/default/registry/main.xml

```diff
@@ -23,18 +25,32 @@
       <element>2.0</element>
     </value>
 
-    <value key="industries">
-      <element>Government</element>
-      <element>Education</element>
-      <element>NGO</element>
-      <element>Corporate</element>
-      <element>Media</element>
+    <value key="industries"
+           purge="false"
+    >
+      <element>gov|Government</element>
+      <element>edu|Education</element>
+      <element>ngo|NGO</element>
```

Comment:
I would spell out this acronym, as "NGO" is not obvious to non-English speakers.

```suggestion
      <element>ngo|Non-government organization (NGO)</element>
```